roles/cyclictest/templates/cyclictestjob.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ meta.name }}-workload-{{ trunc_uuid }}"
+  namespace: "{{ operator_namespace }}"
+spec:
+  template:
+    metadata:
+      labels:
+        app: cyclictest-{{ trunc_uuid }}
+    spec:
+      ttlSecondsAfterFinished: 600
+      containers:
+      - name: cyclictest
+        image: {{ workload_args.image | default('quay.io/cloud-bulldozer/cyclictest:latest') }}
+        command: ["/root/dumb-init", "--"]
+        args: ["/tmp/cyclictest.sh"]
+        imagePullPolicy: Always
+{% if workload_args.nodeselector is defined %}
+      nodeSelector:
+          "{{ workload_args.nodeselector }}"
+{% endif %}
+        resources:
+          requests:
+            memory: {{ workload_args.pod.requests.memory }}
+            cpu: {{ workload_args.pod.requests.cpu }}
+          limits:
+            memory: {{ workload_args.pod.limits.memory }}
+            cpu: {{ workload_args.pod.limits.cpu }}
+        env:
+          - name: DURATION
+            value: "{{ workload_args.duration }}"
+          - name: DISABLE_CPU_BALANCE
+            value: "{{ workload_args.disable_cpu_balance }}"
+          - name: stressng
+            value: "{{ workload_args.stressng }}"
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: cstate
+          mountPath: /dev/cpu_dma_latency
+        - name: cyclictest-volume
+          mountPath: "/tmp/"
+      volumes:
+      - name: cstate
+        hostPath:
+          path: /dev/cpu_dma_latency
+      - name: cyclictest-volume
+        configMap:
+          name: "{{ meta.name }}-workload-{{ trunc_uuid }}"
+          defaultMode: 0555
+      restartPolicy: Never
+      serviceAccountName: benchmark-operator
```

Comment:
I initially had it but removed it because it's not integrated with the wrapper (ES integration) yet anyway.  I was planning to follow up with wrapper integration in a later patchset. Should I still include metadata now?

Reply:
Done